#Do not use Sphinx, as it only needs to make documantation.
# Aug 22nd 2016 by B. Hajgato (Free Unviversity Brussels - VUB)
diff -ru PyBERT-1.7.9.org/PyBERT.egg-info/requires.txt PyBERT-1.7.9/PyBERT.egg-info/requires.txt
--- PyBERT-1.7.9.org/PyBERT.egg-info/requires.txt	2016-06-13 06:15:41.000000000 +0200
+++ PyBERT-1.7.9/PyBERT.egg-info/requires.txt	2016-08-22 11:00:21.297975049 +0200
@@ -4,4 +4,3 @@
 traitsui
 enable
 chaco
-Sphinx
diff -ru PyBERT-1.7.9.org/setup.py PyBERT-1.7.9/setup.py
--- PyBERT-1.7.9.org/setup.py	2016-06-13 06:14:44.000000000 +0200
+++ PyBERT-1.7.9/setup.py	2016-08-22 11:01:00.772975421 +0200
@@ -23,7 +23,6 @@
         'traitsui',
         'enable',
         'chaco',
-        'Sphinx',
         ],
     keywords = ['bert', 'communication', 'simulator'],
     classifiers=[
